packages/lib/src/components/BcmcMobile/BcmcMobile.test.ts

```diff
@@ -20,7 +20,7 @@ describe('BcmcMobile', () => {
     describe('render', () => {
         test('does not render anything by default', () => {
```

Comment:
@sponglord I think the test descriptions are misleading now, as we are rendering the something by default!

Reply:
Doh! I'll fix them!